WalletWasabi/WabiSabi/Client/AliceClient.cs

```diff
@@ -79,6 +79,7 @@ private static async Task<AliceClient> RegisterInputAsync(RoundState roundState,
 			AliceClient? aliceClient;
 			try
 			{
+                coin.CoinJoinInCriticalPhase = false;
```

Comment:
An idea: if the critical phase is very narrow, maybe we can make the critical phase simply longer, SystemAwakeChecker don't really mind, I guess.

Reply:
This is just to make sure that we are starting with the correct values. It should be false already by the time we arrive here.